### ISSUE TITLE
Date Range Picker - dayPickerProps: successfully pass custom 'weekdaysShort' but not 'months'

### ISSUE THREAD

**Issue by sokmeann** [palantir/blueprint#3265]
#### Environment

- __Package version(s)__: "@blueprintjs/datetime": "^3.3.0"
- __Browser and OS versions__: Google Chrome Version 71.0.3578.98 // MacOS Mojave 10.14.2

#### Steps to reproduce

1. Rendering a Date Range Picker via the Redux-Form component `Field`
2. Pass custom `dayPickerProps`: { months: string[], weekdaysShort: string[] }

```
dayPickerProps={{
   months: ["Janvier", "Février", "Mars", "Avril", "Mai", "Juin", "Juillet", "Août", "Septembre", "Octobre", "Novembre", "Décembre"],
   weekdaysShort: ['Di', 'Lu', 'Ma', 'Me', 'Je', 'Ve', 'Sa']
}}
```

#### Actual behavior

Working on an application that needs to support FR-CA and EN-CA. We have added our own language translations (via React Context API). The `weekdaysShort` prop was passed successfully to React-Day-Picker, and the labels were updated correctly. The `months` prop however was not successfully passed, and the component shows the default English labels.

#### Expected behavior

We are expecting both dropdowns for month selection to reflect the custom strings we have passed in the `months` array.

#### Possible solution

Might be an issue with passing the props to React-Day-Picker. Alternatively, allow `months` to be passed on first level props.

Thank you!

**Comment by giladgray** [comment]
@sokmeann hmm interesting this might be a real bug. would you mind investigating and submitting a fix?

**Comment by sokmeann** [comment]
@giladgray I looked through the source code, and the dropdown to select months is rendered by datePickerCaption, which picks the months labels from `localeUtils.getMonths()` uniquely. I wrote a fix to use the `months` prop if available, but I am not able to push to this repo. Do I need to be added as a collaborator?

**Comment by adidahiya** [comment]
https://github.com/gpbl/react-day-picker/pull/843 has been released in react-day-picker v7.3.0, so we should be able to use that version with better type definitions

**Comment by adidahiya** [comment]
fixed & released in datetime 3.9.0
